src/serde_tokenstream.rs

```diff
@@ -98,13 +169,17 @@ struct TokenDe {
 }
 
 impl<'de> TokenDe {
-    fn from_tokenstream(input: &'de TokenStream) -> Self {
+    fn from_tokenstream(
+        span: Option<&DelimSpan>,
+        input: &'de TokenStream,
+    ) -> Self {
         // We implicitly start inside a brace-surrounded struct.
         // Constructing a Group allows for more generic handling.
```

Comment:
```suggestion
        // We implicitly start inside a brace-surrounded struct.
        // Constructing a Group allows for more generic handling.
        // If there is an error at the top level (such as a missing field) it will be attributed to the span of this group which is why we allow users to optionally include this  span for attribution.
```

... or something like that?